modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -1293,4 +1293,41 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  test("-D.. options passed to the child app") {
```

Comment:
We should probably only run those if `TestUtil.isNativeCli` is true. (They'll fail when run with a JVM launcher else, I think)

Reply:
Actually, they seem to run fine there too…

Reply:
Yep, and imo they could be kept, as this PR also affects JVM runners.